src/pages/workspaces/workspace/Dashboard/WorkspaceInformation.ts

```diff
@@ -0,0 +1,40 @@
+import { ReactNode } from 'react';
+import { dl, h } from 'react-hyperscript-helpers';
+import { InfoBox } from 'src/components/InfoBox';
+import {
+  hasProtectedData,
+  hasRegionConstraint,
+  protectedDataMessage,
+  regionConstraintMessage,
+} from 'src/libs/workspace-utils';
+import { InfoRow } from 'src/pages/workspaces/workspace/Dashboard/InfoRow';
+import { InitializedWorkspaceWrapper as Workspace } from 'src/pages/workspaces/workspace/useWorkspace';
+
+const roleString = {
+  READER: 'Reader',
+  WRITER: 'Writer',
+  OWNER: 'Owner',
+  PROJECT_OWNER: 'Project Owner',
+};
+
+interface WorkspaceInformationProps {
+  workspace: Workspace;
+}
+export const WorkspaceInformation = (props: WorkspaceInformationProps): ReactNode => {
+  const { workspace } = props;
+  return dl({}, [
```

Comment:
Nit: props can be omitted if it's empty.

```suggestion
  return dl([
```